### ISSUE TITLE
Update the positioning of Navigation buttons on the Home Page - Mobile Mode

### ISSUE THREAD

**Issue by MA1002643** [MA1002643/theabdullahfolio#4]
The navigation buttons should be positioned at a screen width of 479px and below, exactly as they are on this website: https://next-js-creative-portfolio-website.vercel.app/.
